validator/webui/@polymer/webui-mainpage/webui-mainpage.html

```diff
@@ -298,9 +298,17 @@
 
         // Interprets the parameters after the '#' in the URL.
         var params = getLocationHashParams();
+        // Extracts the code to be validated if sent as a query param.
+        const incomingDocStr = getIncomingDoc(params);
+        if (incomingDocStr) {
+          editor.setEditorValue(incomingDocStr);
+          const format = params['htmlFormat'] || 'AMP4ADS';
```

Comment:
I know the intent is to use this for AMP4ADS but right now the default when not specified in the WebUI is AMP and I'd like to stick to that for consistency.

Reply:
Makes sense. I set the default to `AMP` we will just need to tell partners to include `htmlFormat=AMP4ADS`